slack-api-model-kotlin-extension/src/test/kotlin/test_locally/block/validation/ValidationUtil.kt

```diff
@@ -0,0 +1,27 @@
+package test_locally.block.validation
+
+import java.lang.reflect.Modifier
+import kotlin.test.assertTrue
+
+object ValidationUtil {
+
+    fun validateMethodNames(target: Class<*>, builder: Class<*>) {
+
+        val expectedNames = target
```

Comment:
I think using Java reflection is fine but if you have an alternative idea to implement this test utility, I'd love to know the idea. That said, that's a nice-to-have thing. We can discuss it separately.

Reply:
I'm gonna try a method where we look up classes which have the `@BlockLayoutBuilder` annotation to automatically pull in the builder classes for comparison. If it takes too long or is finnicky I'll give it up.